staff/daniel-ferrandez-latre/cv/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Daniel FL</title>
+</head>
+<body>
+    <main>
+        <header>
+            <h1>Daniel Ferrández Latre</h1>
+        </header>
+        <section>
+            <article>
+                <section>
+                    <ul>
+                        <li>31 años</li>
+                        <li>
+                            <address>
+                                Mare de Deu de la Llum L'Hospitalet
+                            </address> 
+                        </li>
+                        <li>638078908</li>
+                        <li>daniel.sflatre@gmail.com</li>
+                    </ul>
+                </section>
+                <section>
+                    <picture>
+                        <img src="https://avatars2.githubusercontent.com/u/47394894?s=460&v=4" type="">
+                        </picture>
+                    </section>
+                </article>
+            </section>
+
+            <section>
+                <section>
+                    <h2>DATOS ACADÉMICOS</h2>
+                    <article>
+                        <H4>Titulo de grado superior en Desarrollo de Aplicaciones Muliplataforma </H4>
+                        <section>
+                            <p>Cursado en “Centre Educación Roca“ Barcelona.</p>
+                            <p>Conocimientos</p>
+                            <ol>
+                                <li>Html, Css, Xml, Java, Git, JavaScript, Processing, Sql, Java, JSP, PL/SQL</li>
+                                <li> Ubuntu Server, Virtual Machines</li>
+                            </ol>
+                        </section>
+                    </article>
+                    <article>
+                            <H4>Titulo de grado medio de preimpresión</H4>
+                            <section>
+                                <p>Cursado en “Centre Educa# u Roca“ Barcelona.</p>
+                                <p>Conocimientos</p>
+                                <ol>
+                                    <li>Maquetación, fotomontage, diferentes formatos de impresión.</li>
+                                    <li>Programas de la gama adobe (versión CC 2019)</li>
+                                    <ul>
```

Comment:
@DaniLatreCodes and ol or ul tag can only have li as child elements. for nested lists, they should be inside a li tag. please fix.

Reply:
Done!